src/static/riot/competitions/editor/_phases.tag

```diff
@@ -144,6 +203,8 @@
                     self.clear_form()
                 }
             })
+            $('.ui.checkbox', self.root).checkbox()
+            $('.ui.advanced-settings.accordion').accordion()
```

Comment:
probably OK to not have `self.root` on this guy?

Reply:
Adding `self.root` will actually cause this to fail (just confirmed, to make sure I wasn't mis-remembering). I think it has something to do w/ the fact that this is not actually on root, but on one of root's children? I had thought that that didn't matter, since it was still _technically_ a part of root, but for what ever reason, adding `self.root` makes the accordion fail.

Reply:
Agh, probably not a big deal, but would love to not have a global init here -- would a `ref` make sense? `#id`?

Reply:
not sure why i didn't just do a ref in the first place... done now.